### ISSUE TITLE
[OUDS] update the bi-mode wording to dual-mode

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/Orange-Boosted-Bootstrap#3331]
The documentation that describes the "bi-mode" (FR) should be updated in EN to use "dual-mode".
This should also impact the url used to access this page
